crates/sqlexec/src/planner/logical_plan.rs

```diff
@@ -20,7 +66,7 @@ pub enum LogicalPlan {
     Write(WritePlan),
     /// Plans related to querying the underlying data store. This will run
     /// through datafusion.
-    Query(DfLogicalPlan),
+    Datafusion(DfLogicalPlan),
```

Comment:
renamed this because as more of the DDL gets moved in to Extension nodes id expect us to potentially get rid of the DDL variant all together. so calling it `Query` didn't really make sense anymore, as it could potentially be any kind of operation now.